.pre-commit-config.yaml

```diff
@@ -190,6 +190,33 @@ repos:
         files: ^BREEZE.rst$|^breeze$|^breeze-complete$
         pass_filenames: false
         require_serial: true
+      - id: base-operator
+        language: pygrep
+        name: Make sure BaseOperator is imported from airflow.models.baseoperator in core
```

Comment:
I haven't read the mailing list chats yet but why do we have different imports for core vs non-core ?